bindings/python/CMakeLists.txt

```diff
@@ -52,107 +50,192 @@ PROJECT (${PROJECT_NAME} VERSION ${PROJECT_VERSION_STRING} LANGUAGES "C" "CXX")
 
 ################################################################################################################################################################
 
-## Dependencies
+## Helper Functions
+
+# Inspired by https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/655/files
+# Possibly move these generic functions to a PythonBuilding.cmake hosted on base image and use the command INCLUDE (PythonBuilding) in this CMakeLists file
```

Comment:
These macros/functions are not ostk-core specific and are reused in ostk-io, ostk-mathematics, ostk-physics and ostk-astrodynamics. We could potentially create a `PythonBuilding.cmake` file to encapsulate all these functions that are needed to perform the building tasks hereafter

Reply:
Okay for now. Could be part of a next refactor